src/utils/validation.ts

```diff
@@ -12,56 +13,68 @@ export const evmXpubSchema = z.string().regex(/^xpub[a-zA-Z0-9]{107}$/);
 
 export const walletTypeSchema = z.enum(['Provider', 'API3', 'Provider-Sponsor', 'API3-Sponsor']);
 
-export const topUpWalletSchema = z.object({
-  walletType: walletTypeSchema,
-  address: evmAddressSchema.optional(),
-});
-
-export const extendedChainDescriptionSchema = z.object({
-  active: z.boolean(),
-  sponsor: z.string(),
-  topUpWallets: z.array(topUpWalletSchema),
-  updateConditionPercentage: z.number().optional(),
-  airseekerConfig: z
-    .object({
-      deviationThreshold: z.number(),
-      heartbeatInterval: z.number(),
-      updateInterval: z.number(),
-    })
-    .optional(),
-});
-
-export const beaconSchema = z.object({
-  name: z.string(),
-  description: z.string(),
-  logoPath: z.string().optional(),
-  beaconId: evmBeaconIdSchema,
-  airnodeAddress: evmAddressSchema,
-  templateId: evmTemplateIdSchema,
-  chains: z.record(extendedChainDescriptionSchema),
-});
+export const topUpWalletSchema = z
+  .object({
+    walletType: walletTypeSchema,
+    address: evmAddressSchema.optional(),
+  })
+  .strict();
+
+export const extendedChainDescriptionSchema = z
+  .object({
+    active: z.boolean(),
+    sponsor: z.string(),
+    topUpWallets: z.array(topUpWalletSchema),
+    updateConditionPercentage: z.number().optional(),
+    airseekerConfig: z
+      .object({
+        deviationThreshold: z.number(),
+        heartbeatInterval: z.number(),
+        updateInterval: z.number(),
+      })
+      .optional(),
+  })
+  .strict();
+
+export const beaconSchema = z
+  .object({
+    name: z.string(),
+    description: z.string(),
+    logoPath: z.string().optional(),
+    beaconId: evmBeaconIdSchema,
+    airnodeAddress: evmAddressSchema,
+    templateId: evmTemplateIdSchema,
+    chains: z.record(extendedChainDescriptionSchema),
+  })
+  .strict();
 
 export const beaconsSchema = z.record(beaconSchema);
 
-export const secretsSchema = z.object({ filename: z.string(), content: z.string() });
-
-export const airnodeDeploymentSchema = z.object({
-  config: airnodeConfigSchema,
-  secrets: secretsSchema,
-  aws: secretsSchema.optional(),
-});
-
-export const airkeeperDeploymentSchema = z.object({
-  config: airnodeConfigSchema,
-  airkeeper: airkeeperConfigSchema,
-  secrets: secretsSchema,
-  aws: secretsSchema.optional(),
-});
-
-export const deploymentSetSchema = z.object({
-  airnode: airnodeDeploymentSchema,
-  airkeeper: airkeeperDeploymentSchema,
-});
+export const secretsSchema = z.object({ filename: z.string(), content: z.string() }).strict();
+
+export const airnodeDeploymentSchema = z
+  .object({
+    config: z.any(), //airnodeConfigSchema,
```

Comment:
I remember @acenolaza mentioning that you can have a versioned validator?

Reply:
This is just a temporary fix until we've properly sorted that out, to allow for multiple versions of the different config files.

Reply:
This is probably related to https://api3dao.atlassian.net/browse/BEC-430

Reply:
Exactly :)